src/components/shared/ItemCard.vue

```diff
@@ -43,20 +43,15 @@ const props = defineProps({
   link: String
 })
 
-function giveReadingTime(description) {
-  let stringArr = description.split(' ')
-  let adjustment = 0
-  stringArr = stringArr.filter((el) => {
-    if (el.indexOf('<') === -1) {
-      return el
-    }
-    adjustment++
-  })
+function giveReadingTime(text, wordsPerMinute = 200) {
+  // Calculate the number of words in the text
+  const wordCount = text.split(/\s+/).length
 
-  let wordCount = stringArr.length + adjustment
-  const averageWordsPerMinute = 238
+  // Calculate the reading time in minutes
+  const readingTimeInMinutes = wordCount / wordsPerMinute
 
-  return Math.round(wordCount / averageWordsPerMinute)
+  // Round up the reading time to the nearest integer
+  return Math.ceil(readingTimeInMinutes)
```

Comment:
There's nothing wrong with doing this.
This is actually excellent and very welcome!
But this type of refactoring makes the review a little longer as this is outside the scope of the PR.
Good job with that refactoring though.

Reply:
Yes, sorry, I understand, it's because I had several branches open and I didn't want to create a new one, I risked getting lost.
But i get it.